src/Http/Requests/MailCareRequest.php

```diff
@@ -10,13 +10,11 @@ class MailCareRequest extends FormRequest
 {
     public function validator()
     {
-        return Validator::make($this->all(), [
-            'email' => 'required',
-        ]);
+        return Validator::make($this->all(), []);
     }
 
     public function email()
     {
-        return InboundEmail::fromMessage($this->get('email'));
+        return InboundEmail::fromMessage(file_get_contents('php://input'));
```

Comment:
probably this would do as well:

```suggestion
        return InboundEmail::fromMessage($this->post());
```

to get all input